backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -300,6 +300,7 @@ export class TransactionLinkResolver {
         transaction.balanceDate = now
         transaction.decay = decay ? decay.decay : new Decimal(0)
         transaction.decayStart = decay ? decay.start : null
+        transaction.transactionLinkId = contributionLink.id
```

Comment:
```suggestion
        transaction.linkId = contributionLink.id
```

Please correct the field name to `linkId` - this is more clear then that the transaction type defines the context of the linkId (either transactionLinkId or contributionLinkid)

Reply:
@Mogge Please give us feedback on this!

Reply:
After talking to @Mogge the following:
You cannot write the `contributionLink.id` into the field `transaction.transactionLinkId`. You must join on the query  the transaction table with the contribution table. There is the field `contributionLinkId` available.

In the GraphQL Interface you can merge both fields to a `linkId` field and handle it like you did in the client.

Meaning: You can leave the client as it is except the naming of the field - you have to adjust the backend and join with contributions table.